go/libraries/doltcore/schema/alterschema/dropcolumn_test.go

```diff
@@ -27,6 +27,46 @@ import (
 	"github.com/liquidata-inc/dolt/go/store/types"
 )
 
+var TypedRowsSansAge []row.Row
```

Comment:
s/sans/without/g

Get out of here with this francophile nonsense USA! USA!

Reply:
https://lh5.googleusercontent.com/proxy/UAFpmthJ6_r8i644McoXNJ7_kq1hY-q4IdTiHoSp3P2WjvBtP6ZsmA2dhBy7QQO_Jl4YZMteGF1Bf_EpdmBZlGddsCpMAGyeiYJG4eEIxw